pull_request.md

```diff
@@ -0,0 +1,99 @@
+# Pull request
+
+The following outlines how to make changes to a repo. The goals are
+
+- Focused PR's related to specific changes
+- Code review is simple and manageable
+- A clean tree of changes in the commit history
+- Consistency across the commits
+
+We also want to respect the time and effort of those reviewing the work. Keeping PR's small, focused and consistent makes their life easier.
+
+## Always on a branch
+
+No matter how small the change, all changes should be done on a branch and never directly to `master`. This is to support the principle of [Master is always shippable](https://github.com/DEFRA/dst-guides/blob/master/principles/master_is_always_shippable.md)
+
```

Comment:
I realise that this is becoming a bit chicken and egg, but would it be possible to link to the "master is always shippable" principle in this repo (currently in PR #2) rather than to dst-guides?

Reply:
Will do as soon as its merged. I'll hold off merging this one till it is.